src/views/requestAccess/requestAccess.js

```diff
@@ -104,23 +155,54 @@ export const RequestAccess = (props) => {
   }, []);
 
   return (
-    <>
-      <div className='main-container'>
-        <div className="page-title"> Request for Access </div>
-        <div className="sub-title"> CONTACT </div>
-        <InfoField label={"First Name"} changeHandler={setFirstName} info={firstName} />
-        <InfoField label={"Last Name"} changeHandler={setLastName} info={lastName} />
-        {/* <InfoField label={"Phone"} info={data['phone']} /> */}
-        <InfoField label={"Email"} changeHandler={setEmail} info={email} />
-        <hr className="line" ></hr>
-        <div className="sub-title sub-title-padding"> ASSIGN ROLE </div>
-        <RoleDropDown selectionOptions={selectionOptions} selectionHandler={selectionHandler} />
-        <div className="mt-2">
-          <button className="button is-small is-rounded is-primary mx-4" onClick={() => grantAccess(currentSelection, fName, lName, emailAddress, id)} disabled={currentSelection === ""}> GRANT ACCESS </button>
+    <div className='main-container'>
+      <div className="page-title"> Request for Access </div>
+      <div className="sub-title"> CONTACT </div>
+      <InfoField label={"First Name"} changeHandler={setFirstName} info={firstName} />
+      <InfoField label={"Last Name"} changeHandler={setLastName} info={lastName} />
+      {/* <InfoField label={"Phone"} info={data['phone']} /> */}
+      <InfoField label={"Email"} changeHandler={setEmail} info={email} />
+      <hr className="line" ></hr>
+      <div className="sub-title sub-title-padding"> ASSIGN ROLE </div>
+      <RoleDropDown selectionOptions={selectionOptions} selectionHandler={selectionHandler} />
+      <h1 className="section-title">PROPERTY</h1>
+      <div className="typeahead-section">
+        <SearchPanel
+          chips
+          clearLabel="Clear search text"
+          placeholder="Search Properties"
+          small
+          width={400}
+          variant={SearchPanelVariant.radio}
+          choices={propertySearchResults}
+          value={propertySearchText}
+          onSelectionChange={setPropertySelection}
+          onChange={handlePropertySearch}
+          onClear={handlePropertySearch}
+          shadow
+        />
+        <button
+          className="add-property-button"
+          onClick={() => setShowAddProperty(!showAddProperty)}
+          type="button"
+        >
+          <i className="fas fa-plus-circle icon-inline-space"></i>
+        Create New Property
+      </button>
+      </div>
```

Comment:
This property typeahead/selection section makes sense when the drop-down value is "property manager" (which is not currently an option ... why?)

When giving access to JOIN Staff and ADMIN's, these users cannot (should not) be associated with properties

Reply:
I noticed that and wanted to bring it up during a meeting but completely forgot. I'll bring in @htharker42 to see if we need to add Property Manager to the list and then only make this appear then